src/config.ts

```diff
@@ -1,15 +1,33 @@
 import dotenv from "dotenv";
 
+export enum EnvironmentMode {
+  Local = "local",
```

Comment:
Just flagging that I will be introducing an Environment mode config variable with this next feature.

cc @patheard @wmoussa-gc @bryan-robitaille